### DISCUSSION TITLE
Sionna-Ray tracing

### DISCUSSION THREAD

**Discussion by Saman12191** [NVlabs/sionna#132] [Q&A]
Hello,

I am using Sionna ray tracing for a project that involves two transmitters
communicating with one receiver.

Specifically, I am trying to use two transmitters in two different scenes
that communicate with a single receiver in the same location in both scenes.
However, I am facing some issues with the code. I am not sure how to combine
the paths from Tx1 to RX and Tx2 to RX. Additionally, I am having trouble
previewing the different scenes.

Given these challenges, I was wondering if you could offer your expert
opinion on whether the idea of using two scenes for two transmitters is
rational or not. I would be grateful for any insights you can provide.

Thank you in advance for your time and help.

**Comment by jhoydis** [comment]
Hi,

Is there a specific reason why you need two different scenes? You can place multiple transmitters and receivers in a single scene and compute the channels between all transmitters and receivers simultaneously.

Also keep in mind that you can only open a single scene at a time (see the [API Documentation](https://nvlabs.github.io/sionna/api/rt.html#sionna.rt.Scene])). This might cause your difficulties in previewing both scenes.
